lib/commands/plugin/find-plugins.ts

```diff
@@ -24,17 +24,17 @@ export class FindPluginsCommand implements ICommand {
 		}).future<void>()();
 	}
 
-	private printPlugins(plugins: IPlugin[]) {
+	private printPlugins(plugins: IShortInformationPlugin[]) {
 		_.each(plugins, (plugin) => {
 			var pluginDescription = this.composePluginDescription(plugin);
 			console.log(pluginDescription);
```

Comment:
I know this is not part of your changes, but could you replace this console.log with $logger.info. Also we may consider beautifying the output by using Table as @teobugslayer improved output of appstore list command. But for the moment I think using $logger.info will be enough.